### ISSUE TITLE
Anchor Cart Doesn't Accept Ender Pearls

### ISSUE THREAD

**Issue by andrewlalis** [GTNewHorizons/GT-New-Horizons-Modpack#18542]
### Your GTNH Discord Username

____andrew____

### Your Pack Version

2.7.0

### Your Server

private server

### Java Version

Java 21

### Type of Server

Vanilla Forge

### Your Expectation

I expect to be able to insert ender pearls into a Railcraft anchor cart to activate its chunkloading. I expected to be able to do this by interacting with the cart manually, or via an item loader. Neither worked.

### The Reality

I am unable to insert the ender pearl. I cannot shift-click it into the slot, or drop it in. I cannot use any item loader to insert the item.

### Your Proposal

The cart should accept ender pearls as a chunk loading fuel.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by andrewlalis** [comment]
Update on this: apparently the default configuration for the world anchor cart's acceptable fuels in `config/railcraft/railcraft.cfg` doesn't include ender pearls.

I'm still keeping this issue open, because without changing this setting, world anchor carts are completely useless as they only accept the coins which are obtainable only as rewards from certain quests.

**Comment by Ruling-0** [comment]
> Update on this: apparently the default configuration for the world anchor cart's acceptable fuels in `config/railcraft/railcraft.cfg` doesn't include ender pearls.
> 
> I'm still keeping this issue open, because without changing this setting, world anchor carts are completely useless as they only accept the coins which are obtainable only as rewards from certain quests.

Is the anchor's fuel different from the anchor cart? I'm pretty sure it's intended in the default config for the RC anchors to only use the coins as fuel, for the offical servers. QB should mention it.

**Comment by andrewlalis** [comment]
Yes, the anchor itself takes ender pearls, while the cart does not.
